src/Migrator/Factory.php

```diff
@@ -216,7 +216,7 @@ public function runReset(Model $entity, $database, $pretend = false)
 
         $migrator->setConnection($database);
         $migrator->setEntity($entity);
-        $migrator->reset($this->getMigrationPath(), ['pretend' => (bool) $pretend]);
+        $migrator->reset($this->getMigrationPath(), $pretend);
```

Comment:
This should be the only valid changes.

[Migrator::reset](https://github.com/laravel/framework/blob/5.3/src/Illuminate/Database/Migrations/Migrator.php#L228)